tools/lib/src/main.dart

```diff
@@ -28,19 +32,15 @@ void main(List<String> args) {
     }
   }
 
-  const ProcessRunner processRunner = ProcessRunner();
-  final Directory sourceTreeRoot = packagesDir.parent;
-  final File pythonTool = sourceTreeRoot
-      .childDirectory('tools')
-      .childDirectory('tools')
-      .childFile('run_command.py');
-
-  if (!pythonTool.existsSync()) {
-    print('Error: Cannot find ${pythonTool.path}.');
-    io.exit(1);
-  }
+  final CommandRunner<void> commandRunner = CommandRunner<void>(
+      'pub global run flutter_tizen_plugin tools',
```

Comment:
This package is not published so this executable name is somewhat misleading.
```suggestion
      'dart run script/tool/bin/flutter_plugin_tools.dart',
```

Reply:
Great advice, I agree.